feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.spec.ts

```diff
@@ -33,4 +36,38 @@ describe('CustomerTicketingDetailsComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('sohuld prepare content for card', () => {
+    const mockCardContent: Card = {
+      text: ['1'],
+      title: 'ID',
+      customClass: '',
+    };
+    component
+      .prepareCardContent(mockTicketId, 'ID')
+      .pipe(take(1))
+      .subscribe((result) => {
+        expect(result).toEqual(mockCardContent);
```

Comment:
This is awesome where you used take(1). I recall this is referenced using the checkout lib. If you look. at the checkout lib, some includes param of done to include the callback for done(), which helps alleviate async code.

```
it('sohuld prepare content for card', (done) => {
    const mockCardContent: Card = {
      text: ['1'],
      title: 'ID',
      customClass: '',
    };
    component
      .prepareCardContent(mockTicketId, 'ID')
      .pipe(take(1))
      .subscribe((result) => {
        expect(result).toEqual(mockCardContent);
       done();
     });
});
   
```

It's good practice including the done callback to let jasmine know the subscription actually got triggered and we can finish this assertion as without the done callback, it can happen that the test may be 'false positive' in a way that the async code didn't happen to finish, but still the test says it passes.

Make sense?

Reply:
should be applied to any async testing

Reply:
yes, makes sense. I will add it